style.css

```diff
@@ -345,128 +276,124 @@ main {
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 64px 24px;
-  gap: 58px;
-  position: absolute;
-  width: 95vw;
-  height: auto;
+  width: 95%;
 }
 
 .about_me {
   display: flex;
   flex-direction: column;
   align-items: center;
-  gap: 6px;
-  width: 95vw;
+  width: 95%;
   height: auto;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
 }
 
-.illustration-top-container {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  width: 85vw;
+.about-me-illustration-container {
+  position: relative;
 }
 
-.illustration-bottom-container {
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
-  width: 85vw;
+.top-illustration-mobile {
+  position: absolute;
+  top: -30%;
+  left: 85%;
+}
+
+.top-illustration-desktop {
+  display: none;
+}
+
+.bottom-illustration-mobile {
+  position: absolute;
+  bottom: 0;
+  right: 80%;
+}
+
+.bottom-illustration-desktop {
+  display: none;
 }
 
 .about_me_heading {
-  width: 95vw;
+  width: 95%;
   height: auto;
   font-family: "Inter", sans-serif;
   font-style: 700;
-  font-size: 40px;
-  line-height: 52px;
+  font-size: 2.5rem;
+  line-height: 3.3rem;
   text-align: center;
   color: #172b4d;
 }
 
 .about_me_description {
-  width: 95vw;
+  width: 95%;
   height: auto;
   font-family: "Inter", sans-serif;
   font-weight: 400;
-  font-size: 20px;
-  line-height: 32px;
+  font-size: 1.25rem;
+  line-height: 2rem;
   text-align: center;
   color: #42526e;
+  padding-bottom: 1.25rem;
 }
 
 .resume_btn {
   display: flex;
   flex-direction: row;
+  justify-content: center;
   align-items: center;
-  padding: 12px;
-  gap: 10px;
-  width: 159px;
-  height: 48px;
+  padding: 0.75rem;
   background: #36b37f;
   border-radius: 4px;
   border: none;
   font-family: "Inter", sans-serif;
   font-style: normal;
   font-weight: 500;
-  font-size: 17px;
-  line-height: 24px;
+  font-size: 1rem;
+  line-height: 1.5rem;
   text-align: center;
   color: #fff;
   margin: auto;
 }
 
+.resume_btn:hover {
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+  display: flex;
+  align-items: center;
+}
+
+.resume_btn:active {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.7rem;
+  background-color: #008552;
+}
+
 /* Languages, Frameworks and Skills Cards Styling */
 
 /* Languages Card Syles */
 .language_framework_skills {
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 1px;
-  gap: 24px;
   width: 95vw;
   height: auto;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
-  margin-top: 2rem;
+  margin-top: 4rem;
 }
 
 .card {
-  width: 95vw;
+  width: 95%;
   height: 70vh;
```

Comment:
- [ ] I recommend you use a fixed height value in this case because if the user opens the page on a larger screen this card can be distorted and not look professional. Kindly fix that to match the Figma template.
 - Your design:
 - 
<img width="1132" alt="image" src="https://user-images.githubusercontent.com/61919617/206435220-19a458a9-0c54-4251-a926-bd7f03a3d459.png">

 - Figma template:
 - 
<img width="937" alt="image" src="https://user-images.githubusercontent.com/61919617/206435294-7e50d8f2-c4b0-4038-9836-0be6befa147c.png">